support/ci/deploy_website.sh

```diff
@@ -9,12 +8,17 @@ set -eu
 cd www
 ./bin/middleman build
 
-# If we are not on a pull request, on the "auto" branch (which homu uses when
-# auto-merging master), and not on a temporary commit, then run the publish
-# script.
+# If we are not on a pull request, and are on the "auto" branch that
+# homu uses when auto-merging to master, and are not on a temporary
+# commit, then publish to acceptance, and then production/live.
 if [ "${TRAVIS_PULL_REQUEST}" = "false" ] &&
    [ "${TRAVIS_BRANCH}" = "auto" ] &&
    [[ $is_tmp_commit = 1 ]]; then
+  AWS_BUCKET=habitat-www-acceptance ./bin/middleman s3_sync
   ./bin/middleman s3_sync
-  curl -H "Fastly-Key: ${FASTLY_API_KEY}" -X POST "https://api.fastly.com/service/${FASTLY_SERVICE_KEY}/purge_all"
-else echo "Not on master; skipping website deploy"; fi
+  # If we have published, then we need to purge the CDN cache.
+  curl -H "Fastly-Key: ${FASTLY_API_KEY}" \
+       -X POST "https://api.fastly.com/service/${FASTLY_SERVICE_KEY}/purge_all"
```

Comment:
I think this will only purge the live site, since the `FASTLY_SERVICE_KEY` is the part of the URL that changes on the Fastly UI when you switch between the different services.